<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>camera type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCameraType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>camera count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCameraCount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>communication type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCommunicationType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>communication count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCommunicationCount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelIP">
       <property name="text">
        <string>ip:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIP"/>
     </item>
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSerialPortNumber">
     <item>
      <widget class="QLabel" name="labelSerialPortNumber">
       <property name="text">
        <string>serial port number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSerialPortNumber"/>
     </item>
     <item>
      <spacer name="horizontalSpacerSerialPortNumber">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBaudRate">
     <item>
      <widget class="QLabel" name="labelBaudRate">
       <property name="text">
        <string>baud rate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBaudRate"/>
     </item>
     <item>
      <spacer name="horizontalSpacerBaudRate">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutParityBit">
     <item>
      <widget class="QLabel" name="labelParityBit">
       <property name="text">
        <string>parity bit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboParityBit"/>
     </item>
     <item>
      <spacer name="horizontalSpacerParityBits">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDataBit">
     <item>
      <widget class="QLabel" name="labelDataBit">
       <property name="text">
        <string>data bit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboDataBit"/>
     </item>
     <item>
      <spacer name="horizontalSpacerDataBits">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutStopBit">
     <item>
      <widget class="QLabel" name="labelStopBit">
       <property name="text">
        <string>stop bit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboStopBit"/>
     </item>
     <item>
      <spacer name="horizontalSpacerStopBits">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>262</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
